app/connectors/AddressLookupConnector.scala

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import config.FrontendAppConfig
+import connectors.AddressLookupConnector._
+import models._
+import play.api.http.Status._
+import play.api.i18n.Messages
+import play.api.libs.json.{Format, JsError, JsObject, JsSuccess, Json, Reads, __}
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpReads, HttpResponse}
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+import uk.gov.hmrc.http.HttpReads.Implicits._
+
+import scala.util.{Failure, Success, Try}
+
+class AddressLookupConnector @Inject()(http: HttpClient, appConfig: FrontendAppConfig, messages: Messages)(
```

Comment:
I'm getting a runtime exception on this line: No implementation for play.api.i18n.Messages was bound.